_includes/_wallet_install.htm

```diff
@@ -6,30 +6,15 @@
                 <h4 class="text-bold">
                     Install the Gridcoin client
                 </h4>
-                <h4>Windows & Linux</h4>
+                <h4>Wallet Download</h4>
                 <a id="64-bit-windows" class="btn rounded-pill grcbtn m-1" style="display:none"> 
-                    64-bit Windows
+                    Download Gridcoin wallet
                 </a>
-                <a id="32-bit-windows" class="btn rounded-pill grcbtn m-1" style="display:none">
```

Comment:
why is this being removed?

Reply:
It's just linking all platforms to the Gridcoin download page. Previously, it linked to the Github releases page, which is a little more difficult to navigate.

Reply:
If you load this up in Jekyll you'll see how these edits are connected. 

Previously:
Two sections titled "Windows and Linux" and "MacOS" both with buttons that link to Github releases page, which had dozens of things to download and may be difficult for users to understand. Plus, Linux users don't get shown option for PPA

With edits:
One download button which takes user to Download page on Gridcoin.us
Gridcoin.us download page has direct link to download exe for windows,  Direct link to MacOS dmgs, and several options for Linux users depending on their distro. If we can get Linux users to use PPAs, this is better than directly downloading the debs since they will automatically update.